build.gradle

```diff
@@ -53,16 +56,16 @@ dependencies {
                 "work-around may be removed."
     }
     implementation "commons-beanutils:commons-beanutils:1.9.4"
-    implementation "org.springframework.boot:spring-boot-starter-actuator:2.1.8.RELEASE"
-    implementation "org.springframework.boot:spring-boot-starter-web:2.1.8.RELEASE"
-    implementation("org.springframework.data:spring-data-solr:4.1.0.RELEASE") {
+    implementation "org.springframework.boot:spring-boot-starter-actuator:${springBootVersion}"
+    implementation "org.springframework.boot:spring-boot-starter-web:${springBootVersion}"
+    implementation ("org.springframework.data:spring-data-solr:4.1.0.RELEASE") {
```

Comment:
✏️ Remove this space for consistency

Reply:
I didn't manually add it ... any idea where it came from?